CHANGELOG.md

```diff
@@ -1,4 +1,15 @@
 ## 3.14.0 (Unreleased)
+
+FEATURES:
+* **New Data Source:** `google_compute_instance_serial_port` ([#5911](https://github.com/terraform-providers/terraform-provider-google/pull/5911))
+* **New Resource:** `google_compute_region_ssl_certificate` ([#5913](https://github.com/terraform-providers/terraform-provider-google/pull/5913))
+
+IMPROVEMENTS:
+* compute: Add new attribute reference `current_status` ([#5903](https://github.com/terraform-providers/terraform-provider-google/pull/5903))
+* compute: Added `allow_global_access` for to `google_compute_forwarding_rule` resource. ([#5912](https://github.com/terraform-providers/terraform-provider-google/pull/5912))
+* container: added `node_pool.upgrade_settings` ([#5910](https://github.com/terraform-providers/terraform-provider-google/pull/5910))
```

Comment:
Can you clarify whether this is to the `node_pool` block on `google_container_cluster` or a top level field on `google_container_node_pool` (or both)?